app/lib/server/methods/refreshOAuthService.js

```diff
@@ -16,6 +16,6 @@ Meteor.methods({
 
 		ServiceConfiguration.configurations.remove({});
 
-		Settings.update({ _id: /^Accounts_OAuth_.+/ }, { $set: { _updatedAt: new Date() } }, { multi: true });
+		Settings.update({ _id: /^((Accounts_OAuth_)|(SAML_)|(CAS_)|(Blockstack_)).+/ }, { $set: { _updatedAt: new Date() } }, { multi: true });
```

Comment:
the additional parenthesis are not required.. I think removing them makes it easier to read as well
```suggestion
		Settings.update({ _id: /^(Accounts_OAuth_|SAML_|CAS_|Blockstack_).+/ }, { $set: { _updatedAt: new Date() } }, { multi: true });
```